tests/unit/checkboxradio/checkboxradio_core.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * checkboxradio_core.js
+ */
+
+
+(function($) {
+
+module("Checkboxradio: core");
+test("Checkbox", function() {
+	expect( 4 );
+	var input = $("#check"),
+		label = $("label[for=check]");
+	ok( input.is( ":visible" ) );
+	ok( !label.hasClass(".ui-button)") );
+	input.checkboxradio();
+	equal( input.is( ".ui-helper-hidden-accessible.ui-checkboxradio" ), true,
+		"Input has proper classes" );
+	equal( label.is( ".ui-icon-beginning.ui-button.ui-widget.ui-checkboxradio-label.ui-corner-all" ),
+		true, "Label has proper classes" );
+});
+
+test("Radios", function() {
+	expect( 4 );
+	var inputs = $("#radio0 input"),
+		labels = $("#radio0 label");
+	ok( inputs.is(":visible") );
+	ok( labels.is(":not(.ui-button)") );
+	inputs.checkboxradio();
+	ok( inputs.is(".ui-helper-hidden-accessible") );
+	ok( labels.is(".ui-button") );
+});
+
+function assert(noForm, form1, form2) {
+	ok( $("#radio0 .ui-button" + noForm).is(".ui-state-active") );
+	ok( $("#radio1 .ui-button" + form1).is(".ui-state-active") );
+	ok( $("#radio2 .ui-button" + form2).is(".ui-state-active") );
+}
+
+test("radio groups", function() {
+	expect( 12 );
+	$("input[type=radio]").checkboxradio();
+	assert(":eq(0)", ":eq(1)", ":eq(2)");
+
+	// click outside of forms
+	$("#radio0 .ui-button:eq(1)").simulate("click");
+	assert(":eq(1)", ":eq(1)", ":eq(2)");
+
+	// click in first form
+	$("#radio1 .ui-button:eq(0)").simulate("click");
+	assert(":eq(1)", ":eq(0)", ":eq(2)");
+
+	// click in second form
+	$("#radio2 .ui-button:eq(0)").simulate("click");
+	assert(":eq(1)", ":eq(0)", ":eq(0)");
+});
+
+asyncTest( "Checkbox/Radiobutton do not Show Focused State when using Keyboard Navigation", function() {
+	expect( 2 );
+	var check = $( "#check" ).checkboxradio(),
+		label = $( "label[for='check']" );
+	ok( !label.is( ".ui-state-focus" ) );
+	check.focus();
+	setTimeout(function() {
+		ok( label.is( ".ui-state-focus" ) );
+		start();
+	});
+});
+
+// TODO: simulated click events don't behave like real click events in IE
+// remove this when simulate properly simulates this
+// see http://yuilibrary.com/projects/yui2/ticket/2528826 fore more info
```

Comment:
This link is dead (thanks Yahoo). Have you checked that this exclusion is still necessary?